libmscore/keysig.cpp

```diff
@@ -535,6 +566,11 @@ bool KeySig::setProperty(Pid propertyId, const QVariant& v)
                         return false;
                   setShowCourtesy(v.toBool());
                   break;
+            case Pid::KEYSIG_MODE:
+                  if (generated())
+                        return false;
+                  //setMode(v);  // no known conversion from 'const QVariant' to 'Ms::KeyMode'
```

Comment:
KeyMode(v.toInt()) ?

Reply:
error: no matching function for call to 'Ms::KeySig::setMode(int)'

Reply:
oops, indeed